src/mesh/generated/meshtastic/mesh.pb.h

```diff
@@ -205,6 +205,7 @@ typedef enum _meshtastic_HardwareModel {
     meshtastic_HardwareModel_RPI_PICO2 = 79,
     /* M5 esp32 based MCU modules with enclosure, TFT and LORA Shields. All Variants (Basic, Core, Fire, Core2, CoreS3, Paper) https://m5stack.com/ */
     meshtastic_HardwareModel_M5STACK_CORES3 = 80,
+    meshtastic_HardwareModel_SEEED_XIAO_S3 =81,
```

Comment:
This file is automatically generated. This edit should be proposed here instead: https://github.com/meshtastic/protobufs/blob/master/meshtastic/mesh.proto

Reply:
https://github.com/meshtastic/protobufs/pull/587   just added this PR